ui/pages/providers/add/details.js

```diff
@@ -167,6 +177,67 @@ export default function ProvidersAddDetails() {
             <ErrorMessage message={errors.clientsecret} />
           )}
         </div>
+        {kind === 'google' && (
```

Comment:
In a future PR we should consider making separate pages which would be easier to test